assembly/pom.xml

```diff
@@ -83,6 +83,7 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
+      <version>30.1-jre</version>
```

Comment:
had to bump guava to avoid a runtime crash due to backward incompatible API changes.